x-pack/plugins/lens/public/metric_visualization/expression.scss

```
@@ -6,4 +6,16 @@
   width: 100%;
   height: 100%;
   text-align: center;
-}
+
+  .lnsMetricExpression__value {
+    font-size: 60pt;
+    font-weight: 600;
+    border-radius: $euiBorderRadius;
+    padding-left: $euiSize;
+    padding-right: $euiSize;
+  }
+
+  .lnsMetricExpression__title {
+    font-size: 24pt;
+  }
```

Leaving some small SCSS suggestions to utilize more EUI variables and move away from the `pt` unit:

```suggestion
  .lnsMetricExpression__value {
    font-size: $euiSizeXXL * 2;
    font-weight: $euiFontWeightSemiBold;
    border-radius: $euiBorderRadius;
    padding: 0 $euiSize;
  }

  .lnsMetricExpression__title {
    font-size: $euiSizeXL;
  }
```